docs/configuration.rst

```diff
@@ -110,10 +110,9 @@ policy.
        on the same origin.
 
 ``REPORT_PERCENTAGE``
-    Percentage of requests that should see the ``report-uri`` directive.
-    Use this to throttle the number of CSP violation reports made to your
-    ``report-uri``. An **integer** between 0 and 100 (0 = no reports at all).
-    Ignored if ``report-uri`` isn't set.
+    Percentage of requests that should see the ``report-uri`` directive.  Use this to throttle the
+    number of CSP violation reports made to your ``report-uri``. A **float** between 0.0 and 100.0
```

Comment:
> A **float** between 0.0 and 100.0

Won't a float of `100.0` give us a REPORT_PERCENTAGE of 10000.0? Would be good to warn about invalid values maybe?

```
        config["REPORT_PERCENTAGE"] = _REPORT_PERCENTAGE * 100
```

Reply:
I think it'd be nice, to help avoid surprises, but can be a follow-on thing or a ticket for someone in the community to pick up?